src/VCEL.Core/VCEL.Core.csproj

```diff
@@ -1,32 +1,28 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <Authors>Brian Flynn</Authors>
-    <Product>View Client Expression Language</Product>
-    <Description>Spring Expression Like Expression Language</Description>
-  </PropertyGroup>
```

Comment:
This is duplicate of `src/Directory.Build.props`, which applies to all projects.